db_provision.mjs

```diff
@@ -4,13 +4,19 @@
 import { db } from "@vercel/postgres";
 import * as dotenv from "dotenv";
 dotenv.config({
-  path: ".env.development.local",
+  path: ".env.local",
```

Comment:
Why this change here? Did you rename the local environments file from `.env.development.local` to `.env.local`?

Reply:
Yes. The file name is now .env.local

Reply:
changed the name back to .env.development.local